Dockerfile

```diff
@@ -16,4 +16,7 @@ RUN \
   apt-get remove -y git && \
   apt-get -y autoremove
 
-ENTRYPOINT bin/iotagent-json config.js
+RUN npm install pm2@latest -g
+USER node
+ENV NODE_ENV=production
+CMD ["pm2-runtime", "bin/iotagent-json", "config.js"]
```

Comment:
Blank line between ENV and CMD

Reply:
Fixed with c5b4c46e91a2ada6302b8b71d1c0814dd637ef34